src/app/store/entities/sandboxes/utils/create-zip/vue-cli/index.js

```diff
@@ -63,20 +63,22 @@ export default function createZip(
 ) {
   zip.file('README.md', README);
 
-  return zip.loadAsync(files).then(zip => {
```

Comment:
Is there a reason you renamed this from `zip` to `src`?

Reply:
`EsLint: 'zip' is already declared in the upper scope. (no-shadow)`

Reply:
Ah, right 👍